lib/cucumber/multiline_argument/data_table.rb

```diff
@@ -159,8 +159,10 @@ def hashes
       #   [{:foo => '2', :bar => '3', :foo_bar => '5'}, {:foo => '7', :bar => '9', :foo_bar => '16'}]
       #
       def symbolic_hashes
-        @header_conversion_proc = lambda { |h| symbolize_key(h) }
-        @symbolic_hashes ||= build_hashes
```

Comment:
Is this `build_hashes` method used elsewhere? If not I'd delete it. Same with the `@header_conversion_proc` - make sure it isn't used elsewhere before deleting it.

Reply:
- `build_hashes` is called above on line 147 within the method `hashes`, which in turn is called on line 163.
- `@header_conversion_proc` is used elsewhere and the modification to this variable in `@symbolic_hashes` is what is/was causing the issue.

Personally, I wouldn't mind removing usage of `@header_conversion_proc` throughout the file as it seems a bit unnecessary.  However, I think that is outside the scope of this bugfix.

Reply:
Sounds good, thanks for clarifying!